backend/src/test/java/codezap/tag/service/TemplateTagServiceTest.java

```diff
@@ -0,0 +1,271 @@
+package codezap.tag.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+
+import codezap.category.domain.Category;
+import codezap.fixture.CategoryFixture;
+import codezap.fixture.MemberFixture;
+import codezap.fixture.TemplateFixture;
+import codezap.global.ServiceTest;
+import codezap.member.domain.Member;
+import codezap.tag.domain.Tag;
+import codezap.tag.dto.response.FindAllTagsResponse;
+import codezap.tag.dto.response.FindTagResponse;
+import codezap.template.domain.Template;
+import codezap.template.domain.TemplateTag;
+
+
+@Transactional
+class TemplateTagServiceTest extends ServiceTest {
+
+    @Autowired
+    private TemplateTagService templateTagService;
+
+    @Nested
+    @DisplayName("템플릿에 대한 태그 생성")
+    class CreateTags {
+
+        @Test
+        @DisplayName("성공: 템플릿에 해당하는 태그 생성")
+        void createTags() {
+            // given
+            Template template = createTemplate();
+            List<String> tagNames = Arrays.asList("tag1", "tag2", "tag3");
+
+            // when
+            templateTagService.createTags(template, tagNames);
+
+            // then
+            List<String> savedTemplateTagNames = getSavedTagNames(template);
+            assertThat(savedTemplateTagNames).containsExactlyElementsOf(tagNames);
+        }
+
+        @Test
+        @DisplayName("성공: 이미 있는 템플릿 태그가 포함된 경우 중복 저장하지 않고 새 태그만 생성")
+        void createTags_WhenExistTemplateTagContains() {
+            // given
+            Template template = createTemplate();
+            Tag existTag = tagRepository.save(new Tag("tag1"));
+            TemplateTag existTemplateTag = new TemplateTag(template, existTag);
```

Comment:
해당 코드는 없어도 될 것 같은데 만약 필요하다면 다음과 같이 수정해야 할 것 같네요
```suggestion
            TemplateTag existTemplateTag = templateTagRepository.save(new TemplateTag(template, existTag));
```

Reply:
와우 꼼꼼쓰 정꼼꼼으로 개명 어때요?